packages/core/services/signers/BitcoinTrezorDeviceMock.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import EventEmitter from 'events'
+import bip39 from 'bip39'
+import bitcoin from 'bitcoinjs-lib'
+import coinselect from 'coinselect'
+import axios from 'axios'
+
+export default class BitcoinMemoryDevice extends EventEmitter {
+  constructor ({seed, network}) {
+    super()
+    this.seed = seed
+    this.network = network
+    Object.freeze(this)
+  }
+
+  privateKey (path) {
+    return this._getDerivedWallet(path).privateKey 
+  }
+
+  // this method is a part of base interface
+  getAddress (path) {
+    return this._getDerivedWallet(path).getAddress()
+  }
+
+  signTransaction (rawTx, path) { // tx object
+    const txb = new bitcoin.TransactionBuilder.fromTransaction (
+      bitcoin.Transaction.fromHex (txhex), this.network)
```

Comment:
There is no txhex in the TxEntryModel.js. Maybe we should use TxEntryModel.hash here?